src/store/assets/tokens.ts

```diff
@@ -99,9 +99,13 @@ export const mergeAssetBalances = (
 
 const calculateBalanceByType = atomFamily((assetType: string) =>
   atom(get => {
-    const assets: AssetWithMeta[] = get(assetsState);
-    const unanchoredAssets: AssetWithMeta[] = get(assetsUnanchoredState);
-    return mergeAssetBalances(assets, unanchoredAssets, assetType);
+    try {
```

Comment:
just adding a note here -- when you `try/catch` the getters in an atom, if they suspend, it will catch even though it's not an error. the reason why is because for something to suspend, it throws a `Promise`, not an `Error`.